composer.json

```diff
@@ -16,13 +16,15 @@
     "laminas/laminas-cache-storage-implementation": "1.0"
   },
   "require-dev": {
-    "laminas/laminas-cache": "^2.10",
-    "laminas/laminas-cache-storage-adapter-test": "^1.0@dev",
+    "mongodb/mongodb": "^1.8.0",
```

Comment:
I was surprised this requirement wasn't there before: without it no test pass

Reply:
It was required by the travis configuration as there were different versions used for different php versions.